cli/monkey-cam.cpp

```diff
@@ -42,11 +42,26 @@ namespace MonkeyCAM {
 // @TODO: load more parts
 std::unique_ptr<ShapeEndPart> loadEndPart(boost::property_tree::ptree& config) {
   auto type = config.get<std::string>("type");
-  auto endHandle = config.get<double>("end handle");
-  auto transitionHandle = config.get<double>("transition handle");
-  assert(type == "Basic Bezier");
-  return std::unique_ptr<ShapeEndPart> {
-    new BasicBezier { endHandle, transitionHandle } };
+assert(type == "Basic Bezier" || type == "Flat");
+  if(type == "Basic Bezier")
```

Comment:
Follow the coding conventions of the rest of the code here please: space between `if` and `(`, `{` on the same line as the `if` and `else`, etc. Avoid spurious blank lines. 

I'll tend to be a stickler for these kinds of things since I've worked on many projects where consistency hasn't been maintained, and the code becomes a nightmare to read after a while :)